blogs/5_how_to_gorilla.html

```diff
@@ -0,0 +1,502 @@
+                            </li>
+                            <li> Using gorilla CLI to write to files:
+                                <ul>
+                                    <li>Command:
+                                    <br><code>$ gorilla “generate 100 random characters into a file called text.txt”</code>
+                                    </li>
+                                    <li>Output: Using arrow keys, toggle between commands and press <code>Enter</code>
+                                    <br><code>» cat /dev/urandom | env LC_ALL=C tr -dc 'a-zA-Z0-9' | head -c 100 > test.txt </code>
+                                    <br><code>echo $(head /dev/urandom | LC_CTYPE=C tr -dc 'a-zA-Z0-9' | dd bs=100 count=1) > test.txt </code>
+                                    <br><code>dd if=/dev/urandom bs=1 count=100 of=test.txt </code>
+                                    </li>
+                                </ul>
+                            </li>
+                            <li> Other Usage Examples:
+                                <ul>
+                                    <li>Command:
+                                    <br><code>$ gorilla “list all my GCP instances”</code>
+                                    </li>
+                                    <li>Output: 
+                                    <br><code>» gcloud compute instances list --format="table(name,zone,status)"</code>
+                                    <br><code>gcloud compute instances list --format table </code>
+                                    <br><code>gcloud compute instances list --format="table(name, zone, machineType, status)" </code>
+                                    </li>
+                                </ul>
+                            </li>
+                        </ol>
+                    </p>
+                    <h4 id="gorilla-inference"><a href="https://github.com/ShishirPatil/gorilla/tree/main/inference">Gorilla Inference 🦍 </a></h4>
+                    <p>
+                        <i>Note: If you would like to try out zero-shot Gorilla in a playground-like environment check out <a href="https://colab.research.google.com/drive/1DEBPsccVLF_aUnmD0FwPeHFrtdC0QIUP?usp=sharing">this Google Colab</a>. 
+                        </i>
+                        <br>
+                        <br>Running inference with gorilla is now supported on both hosted endpoints and locally! Inference with Gorilla can be done in 3 primary modalities: 
+                            <ol>
+                                <li>CLI inference (Single and Batched Prompt Inference)
+                                </li>
+                                <li>Local Inference with Quantized Models
+                                </li>
+                                <li> <a href="https://github.com/ShishirPatil/gorilla/tree/main/inference#4-private-inference-using-gorilla-hosted-endpoint-on-replicate">Inference with Hosted Endpoint on Replicate</a>
```

Comment:
Might be good to have a pragraph on what this is? Who might want to use this, and also the instructions in brief on how this is done. Can be a short summary of the tutorial on the readme.